recipes/units/all/conanfile.py

```diff
@@ -0,0 +1,30 @@
+import os
+from conans import ConanFile, tools
+
+
+class UnitsConan(ConanFile):
+    name = "units"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/nholthaus/units"
+    description = "A compile-time, header-only, dimensional analysis and unit conversion library built on c++14 with no dependencies"
+    topics = ("unit-conversion", "dimensional-analysis", "cpp14",
+              "template-metaprogramming", "compile-time", "header-only",
+              "no-dependencies")
+    license = "MIT"
+    no_copy_source = True
+
+    _source_subfolder = "source_subfolder"
+
```

Comment:
May be add a `configure()` method here that checks the current c++ standard.
If `self.settings.compiler.cppstd` is defined, check that it is >= 14.
Also check whether `self.settings.compiler.version` >= minimum version of that compiler supporting c++14

Reply:
Thanks. I wasn't exactly sure how to check by compiler which one supports C++14, so instead I check for compiler versions as specified on the unit library's README file.